strax/plugins/plugin.py

```diff
@@ -523,6 +523,10 @@ class IterDone(Exception):
         finally:
             self.cleanup(wait_for=pending_futures)
 
+    def _iter_return(self, chunk_i, **inputs_merged):
```

Comment:
Maybe add a docstring or comment to say this should either return or generate result chunks from the input? (Maybe _iter_compute is an alternative name)